src/TestStack.Seleno/PageObjects/Actions/PageWriter.cs

```diff
@@ -63,8 +63,8 @@ public PageWriter(IElementFinder elementFinder, IComponentFactory componentFacto
 
             var stringValue = GetStringValue(propertyTypeHandling, propertyValue, property.PropertyType);
 
-            _componentFactory.HtmlControlFor<TextBox>(propertyExpression)
-                .ReplaceInputValueWith(stringValue);
+			var ctrl = _componentFactory.HtmlControlFor<TextBox>(propertyExpression);
```

Comment:
Should this change be reverted? It doesn't seem to be doing anything (and the indentation is tabs instead of spaces)?